full-backend-tests/src/test/java/org/graylog2/suggestions/EntitySuggestionsIT.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.suggestions;
+
+import org.graylog.testing.completebackend.apis.GraylogApis;
+import org.graylog.testing.containermatrix.annotations.ContainerMatrixTestsConfiguration;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+@ContainerMatrixTestsConfiguration
+public class EntitySuggestionsIT {
+    private final GraylogApis api;
+
+    public EntitySuggestionsIT(GraylogApis api) {
+        this.api = api;
+    }
+
+    @Test
```

Comment:
The test only starts for me if I apply the following change:
```suggestion
    @ContainerMatrixTest
```